guides/common/modules/proc_salt-viewing-salt-autosign-keys.adoc

```diff
@@ -0,0 +1,16 @@
+[id="salt_guide_viewing_salt_autosign_keys_{context}"]
+= Viewing Salt Autosign Keys
+
+The Salt Keys page lists hosts and their Salt keys.
+You can manually accept, reject, or delete keys:
+
+Use the Salt autosign feature to automatically accept signing requests from managed hosts.
+By default, managed hosts are supplied with a {ManagingHostsDocURL}#salt_guide_salt_keys[Salt key] during host provisioning.
+
+[NOTE]
+====
+This feature only covers the autosign via `autosign.conf` authentication procedure.
+====
+
+.Procedure
+. In the {ProjectWebUI}, navigate to **.
```

Comment:
??

Reply:
What's happening here, friend? We can't really publish this without causing confusion, can we?

Reply:
Added the missing procedure.